versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/cloud-marketplace-payg-integration.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Cloud Marketplace Pay-as-you-go (PAYG) Integration
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/v2.8/integrations-in-rancher/cloud-marketplace/cloud-marketplace-payg-integration"/>
+</head>
+
+Rancher Prime integrates with the [AWS Marketplace](https://aws.amazon.com/marketplace) and [Azure Marketplace](https://azuremarketplace.microsoft.com/) as a pay-as-you-go (PAYG) offering. This brings the value of running and managing Kubernetes environments to cloud customers, who benefit from a new pay-monthly pricing model available through their preferred cloud provider's marketplace. This listing will enable you to manage any CNCF-certified Kubernetes distribution in AWS, Azure, on-prem, or at the edge.
+
+## Limitations
+
+Currently, you must be running Rancher v2.8.4. When you deploy a supported PAYG version, you can update to newer versions of Rancher when the listing is updated.
```

Comment:
Should this be changed to:
```suggestion
We currently recommend running Rancher v2.8.4. When you deploy a supported PAYG version, you can update to newer versions of Rancher when the listing is updated.
```

Reply:
Yeah that will work... Customers deployed with older versions are not required to update, so this verbiage makes sense.